From 5fa8217c9891f427736a78e5c4d10133144a3832 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 21 Jul 2021 17:39:58 +0300
Subject: [PATCH] [SW-50190] gaudi: security: close nic stat/xpcs91

The NIC STAT and XPCS91 along with some additional empty areas need to
be closed as well as the MACs.
Close those areas with the RRs already used near those areas.

Change-Id: I35e34a75388e0300de3c4deaaf280fe89f539bd3
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi/arm/security.c b/arch/gaudi/arm/security.c
index 6113d83..c56b31c 100644
--- a/arch/gaudi/arm/security.c
+++ b/arch/gaudi/arm/security.c
@@ -1418,29 +1418,29 @@
 	max = mmPSOC_TIMESTAMP_BASE;
 	preboot_secure_lbw_rr(LBW_RR_PCIE_MSI_PSOC0, min, max);
 
-	/* PSOC_GPIO0 up to NIC0_STAT: RR(LBW_RR_PSOC1_NIC0_MAC_CH) */
+	/* PSOC_GPIO0 up to NIC0_MAC_CORE_BASE: RR(LBW_RR_PSOC1_NIC0_MAC_CH) */
 	min = mmPSOC_GPIO0_BASE - 1;
-	max = mmNIC0_STAT_BASE;
+	max = mmNIC0_MAC_CORE_BASE;
 	preboot_secure_lbw_rr(LBW_RR_PSOC1_NIC0_MAC_CH, min, max);
 
-	/* NIC0_TS up to NIC1_STAT: RR(LBW_RR_NIC0_TS_NIC1_MAC_CH) */
+	/* NIC0_TS up to NIC1_MAC_CORE_BASE: RR(LBW_RR_NIC0_TS_NIC1_MAC_CH) */
 	min = mmNIC0_TS_BASE - 1;
-	max = mmNIC1_STAT_BASE;
+	max = mmNIC1_MAC_CORE_BASE;
 	preboot_secure_lbw_rr(LBW_RR_NIC0_TS_NIC1_MAC_CH, min, max);
 
-	/* NIC1_TS up to NIC2_STAT: RR(LBW_RR_NIC1_TS_NIC2_MAC_CH) */
+	/* NIC1_TS up to NIC2_MAC_CORE_BASE: RR(LBW_RR_NIC1_TS_NIC2_MAC_CH) */
 	min = mmNIC1_TS_BASE - 1;
-	max = mmNIC2_STAT_BASE;
+	max = mmNIC2_MAC_CORE_BASE;
 	preboot_secure_lbw_rr(LBW_RR_NIC1_TS_NIC2_MAC_CH, min, max);
 
-	/* NIC3_MAC_CH0 up to NIC3_STAT: RR(LBW_RR_NIC3_MAC_CH) */
+	/* NIC3_MAC_CH0 up to NIC3_MAC_CORE_BASE: RR(LBW_RR_NIC3_MAC_CH) */
 	min = mmNIC3_MAC_CH0_BASE - 1;
-	max = mmNIC3_STAT_BASE;
+	max = mmNIC3_MAC_CORE_BASE;
 	preboot_secure_lbw_rr(LBW_RR_NIC3_MAC_CH, min, max);
 
-	/* NIC3_TS up to NIC4_STAT: RR(LBW_RR_NIC3_TS_NIC4_MAC_CH) */
+	/* NIC3_TS up to NIC4_MAC_CORE_BASE: RR(LBW_RR_NIC3_TS_NIC4_MAC_CH) */
 	min = mmNIC3_TS_BASE - 1;
-	max = mmNIC4_STAT_BASE;
+	max = mmNIC4_MAC_CORE_BASE;
 	preboot_secure_lbw_rr(LBW_RR_NIC3_TS_NIC4_MAC_CH, min, max);
 
 	/* mmPSOC_ETR_AXICTL (single register): RR(LBW_RR_ETR) */
